subworkflows/local/coverage_stats.nf

```diff
@@ -0,0 +1,34 @@
+include { MOSDEPTH      } from '../../modules/nf-core/modules/nf-core/mosdepth/main'
+include { SAMTOOLS_VIEW } from '../../modules/nf-core/modules/nf-core/samtools/view/main'
+
+
+workflow COVERAGE_STATS {
+    take: 
+    cram    // channel: [val(meta), path(cram), path(cai)]
+    //bed     // path/to/bed
+    fasta   // path/to/fasta
+    qname   // path/to/qname
```

Comment:
Where are you getting the `qname` file? I didn't think the `blobtoolkit` pipeline uses it.

Reply:
It's an input to `samtools/view` but it's not actually used I just pass `[]` in the main workflow.

Reply:
Might be better not to pass it in the workflow as `[]`, but instead just add it here when you invoke `SAMTOOLS_VIEW`